packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/TestData/Statements/SwitchCaseStatement.txt

```diff
@@ -0,0 +1,17 @@
+// License string
```

Comment:
rename filetype to *.cs and put these into a TestData folder then add a Compile remove tag in the csproj for the test project